irohad/main/application.cpp

```diff
@@ -323,14 +327,14 @@ void Irohad::initOrderingGate() {
       ++local_counter;
       if (local_counter == kMaxLocalCounter) {
         local_counter = 0;
-        if (reject_counter < kMaxDelaySeconds) {
-          reject_counter++;
+        if (reject_delay < kMaxDelay) {
+          reject_delay += std::min(kMaxDelay, std::chrono::milliseconds(1000));
```

Comment:
Could you give a name to the constant of 1k milliseconds? Maybe something like `kDelayIncrement`?